wp-content/themes/access/views/screener-partials/step-2.twig

```diff
@@ -1,31 +1,32 @@
-<fieldset class="{{ styles.fieldset }}" id="step-2" data-track-key="Screener Step: 2"
+<div class="{{ styles.fieldset }}" id="step-2" data-track-key="Screener Step: 2"
   data-track-data='[{"WT.si_n": "ScreeningHH"},{"WT.si_p": "2-Disclaimer"}]'>
   <div class="{{ styles.padding }}">
     <div data-js="step-title" class="{{ styles.number }}">{{ __('Step 2 of 10', 'accessnyc-screener') }}</div>
 
-    <h1 class="{{ styles.heading }}">{{__('What you should know before we begin:', 'accessnyc-screener')}}</h1>
+    <fieldset class="c-question">
+      <legend class="type-h1 w-full {{ styles.heading }}" for="disclaimer-agreement">{{__('What you should know before we begin:', 'accessnyc-screener')}}</legend>
```

Comment:
@theasteve I think the `for="disclaimer-agreement"` is not needed.

Reply:
"for" attributes don't seem to apply to legend tags https://www.w3schools.com/tags/tag_legend.asp